docs/make.jl

```diff
@@ -63,7 +63,7 @@ makedocs(
         "Home" => "index.md",
         "Getting started" => Any[
             "guide/installation.md",
-            "Tutorial" => "guide/tutorial.md",
+            "Getting started" => "guide/tutorial.md",
```

Comment:
Getting started/Getting started is weird, really?

Reply:
Sorry, commits got crossed